### PR TITLE
fix: only update image if it doesn't exist

### PR DESCRIPTION
**Pull request by ndillon1**
#### What's this PR do?
Only set a new artwork url if it didn't previously exist - iOS fix only 

#### Which issue(s) is it related to?
updatePlayback function was updating the image every time it was called which was causing memory leaks

### CODE REVIEW

File: ios/MusicControlManager.m [hunk 1]
```diff
@@ -78,12 +78,13 @@ - (dispatch_queue_t)methodQueue
 
     center.nowPlayingInfo = [self update:mediaDict with:details andSetDefaults:false];
 
-    // Update the image if it exists
-    if ([details objectForKey:@"artwork"] != nil) {
+    // Update the image if it doesn't exists
+    if ([details objectForKey:@"artwork"] == nil) {
```
Comment (tanguyantoine) [inline ios/MusicControlManager.m]: should we do that instead ? 
```obj-c
if ([details objectForKey:@"artwork"] != self.artworkUrl) {
  [self updateNowPlayingArtwork];
}
```

**Comment by tanguyantoine** [comment]
@ndillon1 see my comment. Good catch 👍

File: ios/MusicControlManager.m (hunk 1 above)
Comment (ndillon1) [inline ios/MusicControlManager.m]: yeh that works too, I can update my PR with those changes

Comment (ndillon1) [inline ios/MusicControlManager.m]: @tanguyantoine updated PR with those changes

**Comment by tanguyantoine** [comment]
It looks like you branch is not up to date. could create a new PR with a new branch up to date ?

**Comment by Jeremaiha** [comment]
Could someone here please help me out with instantiating the image url with require?
I tried the following code:
`loadCallToMusicControl() {
    MusicControl.setNowPlaying({
      title: this.title,
      artwork: require('../Images/tempoImage.png')`
But the app crashes.

**Comment by tanguyantoine** [comment]
logs ?

**Comment by Jeremaiha** [comment]
I managed to understand the problem.
First of all it needs to receive a `filePath` and not `RN's require()`, since it doesn't know how to handle it, it checks the prefix if it contains `http\s` so obviously it crashes for `require()`.
After that it checks if a file exist in this path, and here is where I fail to understand why it doesn't find my image in the correct path.

**Pull request merged**